back/app/models/app_configuration.rb

```diff
@@ -136,6 +137,16 @@ def add_missing_features_and_settings
     self.settings = ss.add_missing_settings(settings, schema)
   end
 
+  def sanitize_organization_name
+    organization_name = settings.dig('core', 'organization_name')
+    return unless organization_name.is_a?(Hash) && organization_name.any?
```

Comment:
Can it be anything else than a hash (or `nil`)?

Reply:
It shouldn't be, as it should be a multiloc, and we will be passing it to the `sanitize_multiloc` method if it is is.

Reply:
Then we shouldn't probably test if it's a `Hash`.